.buildkite/pipeline.yml

```diff
@@ -84,26 +85,27 @@ steps:
           manual:
             allowed: true
 
-      # Runs inly on the main branch
-      - label: "Unit tests - MacOS 13 ARM"
-        key: "unit-tests-macos-13-arm"
+  - group: "MacOS tests"
+    key: "macos-unit-tests"
+    steps:
+      - label: "Unit tests - MacOS 15 ARM"
         command: ".buildkite/scripts/steps/unit-tests.sh"
-        branches: main
         artifact_paths:
           - "build/TEST-*.html"
           - "build/TEST-*.xml"
           - "build/diagnostics/*"
           - "coverage-*.out"
         agents:
           provider: orka
-          imagePrefix: generic-13-ventura-arm
+          imagePrefix: generic-base-15-arm-002
         retry:
           manual:
             allowed: true
 
+      # Runs only on main and release branches
       - label: "Unit tests - MacOS 13"
-        key: "unit-tests-macos-13"
         command: ".buildkite/scripts/steps/unit-tests.sh"
+        branches: "main 8.x 8.16 9.*"
```

Comment:
How about 8.* instead? Should prevent us from having to update the branch every release.

Reply:
Good idea!